ABloom/ABloom/Presentation/CoreViews/Menu/SubViews/MyAccountConnecting/ConnectedView.swift

```diff
@@ -9,31 +9,53 @@ import SwiftUI
 
 struct ConnectedView: View {
   @Environment(\.dismiss) private var dismiss
-
   @State var fianceName: String = ""
+  @State var showToast = false
   
   let fianceId: String
+  let myCode: String
   let imageSize: CGFloat = 206
   
   var body: some View {
-    VStack {
+    VStack(spacing: 0) {
+      
       if fianceName == "" {
         ProgressView()
       } else {
         Image("check")
           .resizable()
+          .scaledToFill()
           .frame(width: imageSize, height: imageSize)
+          .padding(.top, 50)
+          .padding(.bottom, 10)
         
         Text("\(fianceName)님과 연결되어 있습니다.")
+          .fontWithTracking(.bodyR, tracking: -0.4)
+          .foregroundStyle(.stone800)
+          .padding(.bottom, 50)
+        
+        myCodeBox
+          .padding(.horizontal, 20)
+        
+        Spacer()
+        
+        if showToast {
+          ToastView(message: "코드가 복사되었습니다")
+        }
+        
+        Spacer()
       }
+      
     }
     .frame(maxWidth: .infinity, maxHeight: .infinity)
+    
     .task {
       guard let fianceName = try? await UserManager.shared.getUser(userId: fianceId).name
       else { return }
       
       self.fianceName = fianceName
     }
+    
     .customNavigationBar {
       Text("상대방과 연결")
```

Comment:
```suggestion
      Text("연결 설정")
```
어제 제이가 디스코드에서 언급해줬던 부분인데, 이 PR에서 같이 수정해주시면 매우매우매우 좋을 것 같습니다!

Reply:
[8f8e5d9](https://github.com/DeveloperAcademy-POSTECH/MacC-Team16-ABloom/pull/151/commits/8f8e5d9659686e3232b380b2648d015d4486977f) 이 커밋에 해당 사항 반영했습니다.